src/gurobi_optimods/min_cut.py

```diff
@@ -170,7 +170,7 @@ def _min_cut_scipy(G, source, sink, create_env):
     ones = np.ones(from_arc.shape)
     data = np.column_stack((ones * -1.0, ones)).reshape(-1, order="C")
 
-    A = sp.csc_array((data, indices, indptr))
+    A = sp.csc_matrix((data, indices, indptr))
```

Comment:
I ran into [Issue in Gurobi v9.x and v10.x: Compatibility issues with numpy 1.25 and scipy 1.11 – Gurobi Help Center](https://support.gurobi.com/hc/en-us/articles/16534198159889-Issue-in-Gurobi-v9-x-and-v10-x-Compatibility-issues-with-numpy-1-25-and-scipy-1-11) here